tests/flow/test_msf.py

```
@@ -0,0 +1,164 @@
+from common import *
+
+GRAPH_ID = "MSF"
+
+class testMSF(FlowTestsBase):
+    def __init__(self):
+        self.env, self.db = Env()
```

Ruff (0.11.9)

`Env` may be undefined, or defined from star imports

(F405)